docs/faq.md

```diff
@@ -114,3 +114,18 @@ For the Example project, run the `EarlGreyExampleSwiftTests` target once then fi
 For physical device builds, replace `Debug-iphonesimulator` with `Debug-iphoneos`.
 
 You can verify the `EarlGrey.framework` is included in the bundle at `EarlGreyExampleSwiftTests.xctest/Frameworks/EarlGrey.framework`
+
+**How should I handle animations?**
+
+Disabling animations improves test performance and eliminates a source of flakiness.
```

Comment:
As per our chat with [Peter](https://github.com/steipete), shouldn't we advice speeding up animations as opposed to disabling it completely so that delegate callbacks and completion blocks are executed in an orderly fashion?